examples/sklearn_autolog/grid_search_cv.py

```diff
@@ -0,0 +1,37 @@
+from pprint import pprint
+
+from sklearn import svm, datasets
+from sklearn.model_selection import GridSearchCV
+
+import mlflow
+from utils import fetch_logged_data
+
+
+def main():
+    mlflow.sklearn.autolog()
+
+    iris = datasets.load_iris()
+    parameters = {"kernel": ("linear", "rbf"), "C": [1, 10]}
+    svc = svm.SVC()
+    clf = GridSearchCV(svc, parameters)
+
+    with mlflow.start_run() as run:
+        clf.fit(iris.data, iris.target)
+
+    # show data logged in the parent run:
+    print("parent run")
+    for key, data in fetch_logged_data(run.info.run_id).items():
+        print("\n---------- logged {} ----------".format(key))
+        pprint(data)
+
+    # show data logged in the child runs
+    print("\nchild runs")
```

Comment:
Nit: Can we make `parent run` and `child_runs` stand out more? E.g.:

```
====== parent run ======
...
====== child runs ======
```

Reply:
Done!